components/resourceDetails/resourceSharing/agentAccessList/index.test.jsx

```diff
@@ -82,7 +82,7 @@ describe("AgentAccessList", () => {
   useFetchProfile.mockReturnValue({ webId });
 
   it("renders loading while loading permissions for access control", () => {
-    usePermissions.mockReturnValue({ permissions: null });
+    usePermissions.mockReturnValue({ permissions: [] });
```

Comment:
AgentAccessList (the component) has this line: 

L44:  const loading = !permissions;

which checks is permissions is null, which is causing this snapshot to fail. We need to update that to check for length.